examples/network/events/interactionEvents.html

```diff
@@ -87,6 +87,15 @@
         console.log('dragEnd Event:', params);
         console.log('dragEnd event, getNodeAt returns: ' + this.getNodeAt(params.pointer.DOM));
     });
+    network.on("controlNodeDragging", function (params) {
+        params.event = "[original event]";
+        document.getElementById('eventSpan').innerHTML = '<h2>control node dragging event:</h2>' + JSON.stringify(params, null, 4);
+    });
+    network.on("controlNodeDragEnd", function (params) {
+        params.event = "[original event]";
+        document.getElementById('eventSpan').innerHTML = '<h2>control node drag end event:</h2>' + JSON.stringify(params, null, 4);
+        console.log('controlNodeDragEnd Event:', params);
```

Comment:
when I visit the [events/interactionEvents.html](http://127.0.0.1:8080/examples/network/events/interactionEvents.html) example to test this, and I drag a node then release it, I don't see the `controlNodeDragEnd Event:` logged out.

@vasa93 curious, could you tell me what I am missing?  what do we need to test the functionality in this PR?

![screen shot 2018-09-05 at 11 43 31 am](https://user-images.githubusercontent.com/2119400/45114185-2c3bcb80-b101-11e8-8974-0b2215097dfd.png)

Reply:
here are the testing steps I tried in list and gif form:

1) visit example http://127.0.0.1:8080/examples/network/events/interactionEvents.html
2) open browser console
3) drag a node
4) look for `controlNodeDragEnd Event:` in the browser console 

![vis-pr-3771-after](https://user-images.githubusercontent.com/2119400/45114385-b6842f80-b101-11e8-8e7b-d80cd27fede3.gif)